resources/assets/uad_lists.json

```
@@ -5204,7 +5204,7 @@
   {
     "id": "com.samsung.android.unifiedprofile",
     "list": "Oem",
-    "description": "My Profile\nRelated to Samsung Members?\n",
+    "description": "My Profile\n\nRelated to Samsung Members?\n",
```

```suggestion
    "description": "My Profile\nRelated to Samsung Members?",
```
No need for a double whiteline here. Single is enough (just like everywhere else).